install/gcp/installer/forseti_server_installer.py

```diff
@@ -101,16 +106,19 @@ def deploy(self, deployment_tpl_path, conf_file_path, bucket_name):
             instance_name = 'forseti-{}-vm-{}'.format(
                 self.config.installation_type,
                 self.config.identifier)
-            self.wait_until_vm_initialized(instance_name)
 
             # Create firewall rules.
             self.create_firewall_rules()
+            self.wait_until_vm_initialized(instance_name)
```

Comment:
Why is this moved?

Reply:
The service account rule to ssh into the VM needs to be created so VM initialization is successful.

Reply:
Ah, that makes sense.  Great move!